beater/server.go

```diff
@@ -80,7 +80,7 @@ type handler func(w http.ResponseWriter, r *http.Request)
 
 func createHandler(p processor.Processor, config Config, publish successCallback) handler {
 	return func(w http.ResponseWriter, r *http.Request) {
-		logp.Debug("handler", "Request - URI: %s ; Method: %s", r.RequestURI, r.Method)
+		logp.Debug("handler", "Request - URI: %s ; Method: %s; Content Length: %d", r.RequestURI, r.Method, r.ContentLength)
```

Comment:
I think we should use the same format as `metrics.go` in beats (`field=value`):

    2017/08/24 13:56:45.531578 metrics.go:39: INFO Non-zero metrics in the last 30s: beat.memstats.memory_alloc=-24968 beat.memstats.memory_total=21488 libbeat.pipeline.events.retry=1